build.sbt

```diff
@@ -43,15 +43,44 @@ lazy val datasource = project.settings(
   libraryDependencies ++= dbDependencies
 )
 
+lazy val libs = (akkaDependencies ++ testDependencies ++ jsonDependencies ++ testContainers) :+ cats
+
 lazy val portal = project
   .configs(IntegrationTest)
   .settings(
     name := "Portal",
     commonSettings,
     Defaults.itSettings,
-    libraryDependencies ++= akkaDependencies ++ testDependencies ++ jsonDependencies ++ macwire ++ testContainers,
-    libraryDependencies += cats,
+    libraryDependencies ++= libs,
     //TODO need to check out parallel execution
     addCommandAlias("testAll", "; test ; it:test")
   )
-  .dependsOn(datasource)
+  .aggregate(repositories, services, controllers, utils)
+  .dependsOn(
+    datasource,
+    repositories % "compile->compile;test->test",
+    services,
+    controllers,
+    utils % "compile->compile;test->test"
+  )
+
+lazy val utils =
+  (project in file("utils")).settings(commonSettings, libraryDependencies ++= libs).configs(IntegrationTest)
+
+lazy val repositories =
+  (project in file("repositories"))
+    .settings(libraryDependencies ++= libs)
```

Comment:
We should choose dependencies more specifically instead of adding all possible libraries

Reply:
Done